DEMO.md

```diff
@@ -0,0 +1,189 @@
+# DEMO
```

Comment:
Ok it is Friday afternoon and it is also beer o'clock but I was thinking to convert this file (and filters too) as a jupyther notebook and execute it as part of CI job during PR checks to making sure it is still working 😃 
It is mainly a set of curl/REST calls to be converted as Python code :)
Wdyt? 
![image](https://user-images.githubusercontent.com/244607/134683867-afb1d16b-cf78-4d6d-82b8-806ba058ddd7.png)

Reply:
Makes sense, I've created https://issues.redhat.com/browse/MGDOBR-68